tests/test_integration/test_site_resource.py

```diff
@@ -0,0 +1,43 @@
+from tests.test_integration.test_case import YolaServiceTestCase
+
+from tests.test_integration.helpers import create_site, create_user
+
+
+class TestYolaSite(YolaServiceTestCase):
+    """Yola: Site resource"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestYolaSite, cls).setUpClass()
+        cls.site = create_site(cls.service)
+
+    def test_can_list_sites(self):
+        sites = self.service.list_sites(page_size=999)['results']
```

Comment:
Without it, I was getting bad failures after several test runs forced my result to not be on the first page. Though this isn't really a fix. We should address this pagination problem at some point.